### PR TITLE
Extra error message when disk not bootable during import

### PR DESCRIPTION
**Pull request by zoran15**
Extra info in translate timeout due to disk potentially not being bootable in GCE.

### CODE REVIEW

File: daisy_workflows/image_import/debian/translate_debian.wf.json [hunk 1]
```diff
@@ -77,7 +77,8 @@
           }
         }
       ],
-      "Timeout": "60m"
+      "Timeout": "60m",
+      "ErrorDescription": "This might indicate the disk is not bootable in Google Compute Engine. Make sure the disk satisfies the requirements documented at https://cloud.google.com/compute/docs/import/importing-virtual-disks."
```
Comment (dntczdx) [inline daisy_workflows/image_import/debian/translate_debian.wf.json]: is it possible to catch the bootable issue other than waiting for timeout? I think response of instance.start API should be able to give us some info about the boot.

Comment (zoran15) [reply daisy_workflows/image_import/debian/translate_debian.wf.json]: We don't call instance.start, instances are auto started on create. 

Also, instance.create and instance.start only return errors, not detailed status on whether OS has booted. GCE doesn't know if OS has booted or not or if there's an OS at all running on the instance.

File: daisy/step.go [hunk 2]
```diff
@@ -37,6 +38,8 @@ type Step struct {
 	name string
 	w    *Workflow
 
+	//Run error description
+	ErrorDescription string `json:",omitempty"`
```
Comment (adjackura) [inline daisy/step.go]: TimeoutErrorDescription, TimeoutErrorText, or just TimeoutDescription/TimeoutText? The comment should be updated to reflect that this is just used for time out and not other errors

Comment (zoran15) [reply daisy/step.go]: Renamed to TimeoutDescription. Also, changed the actual description based on Shawnique's suggestion.

**Pull request merged**